cmd/traffic_cache_tool/CacheDefs.h

```diff
@@ -42,6 +42,25 @@ using namespace ApacheTrafficServer;
 
 namespace ts
 {
+#define dir_clear(_e) \
```

Comment:
Don't use `#define` unless absolutely required. Use inline functions. That avoid argument evaluation issues and bizarre things like the `do` / `while(0)` construct.

Reply:
Some of the functions here are the exact implementation of ATS for debugging purposes. I will reimplement these after I am done tracking the outstanding bugs.